src/Page/HelpSelfSingle.elm

```diff
@@ -71,37 +73,46 @@ view slug model =
 
 renderResourceList : Model -> List CategoryResource -> Html Msg
 renderResourceList model resources =
-    ul []
+    ul [ css [ resourceListStyle ] ]
         (List.map
             (\resource ->
-                li []
-                    ([ h2 []
-                        [ button [ onClick (ToggleResource resource.title) ] [ text (t resource.title) ]
+                if isExpanded model resource.title then
+                    li [ css [ expanderItemStyle ] ]
+                        [ button [ onClick (ToggleResource resource.title), css [ expanderButtonStyle ] ]
+                            [ h2 [ css [ expanderHeadingStyle ] ] [ text (t resource.title) ]
+                            , span [ css [ expanderSymbolStyle, rotate90Style ] ] [ text ">" ]
+                            ]
+                        , div [ css expanderDefinitionStyle ]
+                            ([]
+                                ++ renderResourceDetails resource
+                            )
+                        ]
+
+                else
+                    li [ css [ expanderItemStyle ] ]
+                        [ button [ onClick (ToggleResource resource.title), css [ expanderButtonStyle ] ]
+                            [ h2 [ css [ expanderHeadingStyle ] ] [ text (t resource.title) ]
+                            , span [ css [ expanderSymbolStyle ] ] [ text ">" ]
+                            ]
                         ]
-                     ]
-                        ++ (if isExpanded model resource.title then
-                                renderResourceDetails resource
-
-                            else
-                                [ text "" ]
-                           )
-                    )
             )
             resources
         )
 
 
 renderResourceDetails : CategoryResource -> List (Html msg)
 renderResourceDetails resource =
-    renderQuotes resource.quotes
-        ++ [ p [] [ text (t resource.summary) ]
-           , a [ href (t resource.linkHref) ] [ text (t resource.linkName) ]
-           ]
+    [ blockquote [] ([] ++ renderQuotes resource.quotes)
```

Comment:
I think the blockquote needs to be around each one since they are not same person.